### PR TITLE
doc(website): Adding page for creating new Java plugin

### PR DESCRIPTION
**Pull request by mnorbury**

### CODE REVIEW

File: website/pages/docs/developers/creating-new-plugin/java-source.mdx [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+
+## Apache Arrow Java SDK
+
+Like other CloudQuery SDK the Java SDK uses [Apache Arrow] as part of the underlying CloudQuery type system.
+
+## Creating Your First Plugin
+
+As the Java SDK is still new, we don't yet have a plugin scaffold generator. For now, we recommend starting by copying the code from an [existing Java plugin](https://github.com/cloudquery/cloudquery/tree/main/plugins/source/bitbucket).
+
+<!-- vale off -->
```
Comment (erezrokah) [inline website/pages/docs/developers/creating-new-plugin/java-source.mdx:48]: In `mdx` files this needs to be `{/* <!-- vale off --> */}` (noticed the `{}`), or you can change this file to `md` if you don't use any React components

Comment (mnorbury) [reply website/pages/docs/developers/creating-new-plugin/java-source.mdx]: Would this be why Vercel is failing?

Comment (erezrokah) [reply website/pages/docs/developers/creating-new-plugin/java-source.mdx]: Yes probably since it breaks MDX syntax

**Comment by erezrokah** [comment]
You can also add words to https://github.com/cloudquery/cloudquery/blob/main/.github/styles/Vocab/Base/accept.txt to get the linter to know them

**Review by hermanschaaf — APPROVED** [review]
Looks great!

**Review by erezrokah — APPROVED** [review]

**Pull request merged**